components/camel-jms/src/test/java/org/apache/camel/component/jms/JmsRequestReplyTempQueueMultipleConsumersTest.java

```diff
@@ -45,7 +45,7 @@
 public class JmsRequestReplyTempQueueMultipleConsumersTest extends CamelTestSupport {
 
     @RegisterExtension
-    public ArtemisService service = ArtemisServiceFactory.createPersistentVMService();
+    public ArtemisService service = ArtemisServiceFactory.createVMService();
```

Comment:
Is-this change required?

Reply:
The test behave the same, the persistence part is not used anyway, the VMService is a little bit faster, but I can restore PersistentVM

Reply:
I think using the non-persistent service is fine on this case. I don't think the broker needs to persist messages in this scenario.

Reply:
The question is more: is it an integration or a unit test? If it is a unit test, this change makes sense otherwise in the case of an integration test, it is better to keep the persistence even if it is faster.

Reply:
This is a unit test. It is named as so (our integration tests are named as `*IT.java`).